### PR TITLE
Arin ship health

### PR DESCRIPTION
**Pull request by reinscha**
Added health to the ships, updated tests as well

### CODE REVIEW

File: src/main/java/cs361/battleships/models/Result.java [hunk 1]
```diff
@@ -2,9 +2,10 @@
 
 public class Result {
 
+	AtackStatus status = AtackStatus.INVALID;
```
Comment (PembleC) [inline src/main/java/cs361/battleships/models/Result.java]: I already did this in my Result class, but its alright.

File: src/main/java/cs361/battleships/models/Ship.java [hunk 2]
```diff
@@ -9,6 +9,7 @@
 public class Ship {
 
 	private int size;
+	private int HP;
```
Comment (PembleC) [inline src/main/java/cs361/battleships/models/Ship.java]: You might want to a remove health function

File: src/test/java/cs361/battleships/models/BoardTest.java [hunk 3]
```diff
@@ -11,4 +12,41 @@ public void testInvalidPlacement() {
...
+    }
+    @Test
+    public void testShipHealthMineSweeper () {
+        Ship ship = new Ship("Minesweeper");
+        assertEquals(ship.getHealth(), 2);
+    }
+    @Test
+    public void testShipHealthDestroyer() {
+        Ship ship = new Ship("Destroyer");
+        assertEquals(ship.getHealth(), 3);
```
Comment (PembleC) [inline src/test/java/cs361/battleships/models/BoardTest.java]: Simple but good

**Comment by poliakosOSU** [comment]
These changes look good

**Comment by PembleC** [comment]
Right On!

**Pull request merged**